### ISSUE TITLE
Summernote Fields on Admin stopped working from one day to another.

### ISSUE THREAD

**Issue by goromachine** [summernote/django-summernote#379]
Hi, I've been using this package for few months without any problem, but three weeks ago I've noticed that the same code that I had working was not working on my admin panel.
The code is:
```
class ProjectAdmin(SimpleHistoryAdmin, ImportExportModelAdmin, SummernoteModelAdmin, AdminVideoMixin):
    resource_class = ProjectResource
    list_display = (
        'name', 'user', 'status', 'category', 'likes', 'creation_date', 'is_deleted')
    list_filter = (
        'user', 'category', 'status', 'is_deleted')
    summernote_fields = ('description',)

    def get_readonly_fields(self, request, obj=None):
        if obj:  # editing an existing object
            return self.readonly_fields + ('creation_date', 'likes')
        return self.readonly_fields
```
I did not change this code at all and it was working perfectly on package version `0.8.10.0`. Now to check, I updated to the latest version `0.8.11.4` to check but this didn't solved it.

Additionally because of the version upgrade, now I have this on my templates too:
![image](https://user-images.githubusercontent.com/12257503/68941460-3a29b980-07a6-11ea-9539-89d4611f680e.png)

This is how it looks on admin panel + the console error on the browser.
![image](https://user-images.githubusercontent.com/12257503/68941587-9260bb80-07a6-11ea-9355-97c2905a8573.png)

EDIT:
- The admin problem is happening on local and server.
- The second error on the template for now is on local because I do not uploaded to server yet.

**Comment by DiemenDesign** [comment]
If you didn't change your code, before you upgraded the version of Summernote, it's possible that if your browser was updated that something there changed. Have you tried different browsers so we can tell if it's a Summernote things that are changed or something else?

**Comment by goromachine** [comment]
Tried with Safari, Chrome and Firefox on MAC and Chrome on Windows 10. All on the latest version and the problem is exactly the same.

**Comment by goromachine** [comment]
I would like to know if anyone had the same problem or how could I try to solve this. is affecting my development environment directly. My users already reported the bug too. Any advice on what should I look first to try to solve this?

**Comment by DiemenDesign** [comment]
Have you tried using Summernote on a standalone page, just to check if the issues persist? I ask if you could do this, to make sure some other script you may be using may be interfering with Summernote. At least that may give you a clue whether it's Summernote or another script causing issues. Also, you could remove or comment out the references to other loaded scripts one by one and see if one or more is the problem.

**Comment by goromachine** [comment]
Same problem happening on a standalone page.

**Comment by goromachine** [comment]
Does anyone had the same problem?

**Comment by goromachine** [comment]
Despite this happened without doing any change on my code, I solved it by setting `'iframe':True` on settings.py.
The SummernoteInplaceWidget do not work for Admin.
Then, about the problem that appeared when I upgraded to the latest version of the package, I just had to use `SummernoteInplaceWidget`in front of the already working fine `SummernoteWidget`.
What I don't know about the Inplace widget is if this is how it's supposed to look:
![image](https://user-images.githubusercontent.com/12257503/70045570-9e30e800-15c4-11ea-98c7-9d84e3b7cd68.png)

If not, how am I supposed to apply a different style? any examples? 
Thank you.
